docs/tutorials/subscription-platform.md

```diff
@@ -289,6 +289,141 @@ After the initial payment during subscription creation, the recurring payments f
 
 The script will make up to a configurable number of [attempts to pay](https://developer.paypal.com/docs/archive/express-checkout/ec-set-up-reference-transactions/#capture-future-payments) an invoice before cancelling the subscription.  This attempts count is saved to the invoice itself as metadata.  The invoice's metadata is also used to prevent sending multiple failed payment emails per invoice from the PayPal payment attempts.
 
+## Creating test IAP subscriptions locally
+
+The Subscription Platform supports RP integrations with Google and Apple IAP (In-App Purchases). Below are some instructions for developing these features locally in FxA.
```

Comment:
Note to self to add in next revision:

These test subscriptions are not true IAP subscriptions (i.e. there is no record of them in Apple or Google's DBs), but rather, they are representations in SubPlat's Firestore DB which we use internally.

In reality, an RP Android or iOS app would register a new IAP subscription by hitting an auth server endpoint and subscription changes would come from Real-time developer notifications for Google Play subscriptions and App Store Server notifications for App Store subscriptions.